packages/techdocs-common/src/stages/generate/techdocs.ts

```diff
@@ -42,7 +42,7 @@ export class TechdocsGenerator implements GeneratorBase {
    * The default docker image (and version) used to generate content. Public
    * and static so that techdocs-common consumers can use the same version.
    */
-  public static readonly defaultDockerImage = 'spotify/techdocs:v0.3.2';
+  public static readonly defaultDockerImage = 'spotify/techdocs:0.3.3';
```

Comment:
Was omitting `v` from the tag intentional? Can cause confusion perhaps? :)
https://hub.docker.com/r/spotify/techdocs/tags?page=1&ordering=last_updated

Reply:
It's okay to omit it though and be consistent with that pattern for future.

Reply:
Sorry for all the nitpicking, the name of the release has `v` in it, but the tag doesn't. https://github.com/backstage/techdocs-container/releases/tag/0.3.3

Reply:
Oh yes that should be consistent. Thanks for pointing this out @OrkoHunter, I believe I can edit the tag, maybe I need to release again. Let me check!

Reply:
Updated @OrkoHunter ✅